### DISCUSSION TITLE
Scheduled reminder does not exclude draft PRs

### DISCUSSION THREAD

**Discussion by bensampaio** [community/community#40523] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have [this scheduled reminder](https://github.com/orgs/catawiki/teams/fe/settings/reminders/37255) which is configured to ignore draft PRs but they don't seem to be excluded anyway. The [PR in question](https://github.com/catawiki/cw-buyer/pull/4727) got some reviews already but was moved back to draft. Could this be the cause of the issue? Even if that's the cause, shouldn't it be hidden/ignored?

**Comment by ossanna16** [comment]
Hi there @bensampaio and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
